apps/condo/domains/billing/schema/BillingIntegration.js

```diff
@@ -107,16 +201,24 @@ const BillingIntegration = new GQLListSchema('BillingIntegration', {
             defaultValue: false,
         },
 
-        // TODO(DOMA-1647): Need better solution, used to test UPS flow for now
         isHidden: IS_HIDDEN_FIELD,
-
-        displayPriority: DISPLAY_PRIORITY_FIELD,
-        label: LABEL_FIELD,
-        gallery: GALLERY_FIELD,
-        price: PRICE_FIELD,
     },
     hooks: {
-        afterChange: logoMetaAfterChange,
+        afterChange: async (args) => {
+            await logoMetaAfterChange(args)
+            await bannerPromoMetaAfterChange(args)
+        },
+        validateInput: ({ resolvedData, addValidationError, existingItem }) => {
+            const newItem = { ...existingItem, ...resolvedData }
+            if ((newItem['instruction'] && newItem['setupUrl']) ||
+                (!newItem['instruction'] && !newItem['setupUrl'])) {
+                addValidationError(BILLING_INTEGRATION_SINGLE_CONNECT_WAY_ERROR)
```

Comment:
We are about to use `throw new GQLError()` instead. Are there reasons to use `addValidationError()`?

Reply:
It's not working in fields error, has awful DX, and nobody except tech. support read it. Should I continue?

Reply:
😿 
What about keeping "old style" errors within field hooks?